db/db.ts

```diff
@@ -3,13 +3,12 @@ import { config as dotenvConfig } from "dotenv-safe"
 
 dotenvConfig()
 
-const dbServer: string | undefined = process.env.DB_SERVER
-const dbUser: string | undefined = process.env.DB_USER
-const dbPassword: string | undefined = process.env.DB_PASSWORD
-const dbName: string | undefined = process.env.DB_NAME
+const host: string = process.env.HOST_IP || 'localhost';
+const port: number = parseInt(process.env.PORT || '27017');
+
+const client: MongoClient = new MongoClient(`mongodb://${host}:${port}`);
```

Comment:
Why do you need distinction between host and port?

Reply:
Maybe I've got influenced by documentation. Peharps it could be a single string.

![image](https://github.com/GeoSales-Evolution/multi-tenant-uploader/assets/38257122/fc68f596-b848-422d-9985-18dfd4b75f70)


![image](https://github.com/GeoSales-Evolution/multi-tenant-uploader/assets/38257122/ddf873e3-ac18-4224-8af2-d4ef09a19748)

Reply:
It makes sense in a CLI, but note that when supplying an URL it has all information, but when supplying host and port it didn't supplied url schema.